packages/thirdweb/src/transaction/read-contract.ts

```diff
@@ -111,7 +111,7 @@ export type ReadContractOptions<
  *
  * const { data, isLoading } = useReadContract({
  *   contract,
- *   method: "function tokenURI(uint256 tokenId) returns (string)"}),
+ *   method: "function tokenURI(uint256 tokenId) returns (string)"),
```

Comment:
same here

Reply:
good bot :D

Reply:
Ser. Still wrong here

Reply:
hmm? i updated the code ser

Reply:
There's still an extra ')' in there at the end no?

Reply:
I think the bot comment got us confused. I did update this part, can no longer see the extra ")" in the code

Reply:
_the comment should resolved on itself, but since we are using this thread maybe thats why it did_

Reply:
Oh yeah that must be it. Looks fine on gh